spec/method-param-decorator.test.ts

```diff
@@ -42,9 +42,10 @@ const app = new DanetApplication();
 
 Deno.test('@Res and @Query decorator', async () => {
 	await app.init(MyModule);
-	app.listen(3000);
+	const port = Math.round(Math.random() * 10000);
+	app.listen(port);
 
-	const res = await fetch('http://localhost:3000?myvalue=foo', {
+	const res = await fetch(`http://localhost:${port}?myvalue=foo`, {
```

Comment:
Picking a random port can randomly fail as there are no guaranty it is open but port 0 can be used to get a random open port.
Using an event listener on `listen` event, we can then retrieve the port.

Something along those lines should do (untested):
```suggestion
	const serverStarted = new Promise(resolve => app['app'].addEventListener('listen', ev => resolve(ev)));
	app.listen(0);
	const { port } = await serverStarted;
```

It's a bit ugly because there is currently no exposed way to hook on the `listen` event without accessing `app.app` like this, but that's easy to solve by exposing `app.app.addEventListener` or by changing `app.listen` to return the listen event (which contain the port) as a promise.